jlm/llvm/opt/alias-analyses/PointerObjectSet.hpp

```diff
@@ -876,10 +920,10 @@ class PointerObjectConstraintSet final
    * Finds a least solution satisfying all constraints, using the Worklist algorithm.
    * Descriptions of the algorithm can be found in
    *  - Pearce et al. 2003: "Online cycle detection and difference propagation for pointer analysis"
-   *  - Hardekopf et al. 2007: "The Ant and the Grasshopper".
+   *    - Online Cycle Detection
```

Comment:
Indentation

Reply:
This is a sublist, since Difference Propagation is a technique described in the paper. I changed the comment to make it clearer.